recipes/museval/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "museval" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2e1334dd3367dea562c1902515f6d55e87dc3072cfc411c616ac6f0a48d31d5f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py<36 or py38]
```

Comment:
Can this be `noarch: python`?

Reply:
Not at the moment, as the dependency `musdb` as the same target requirements. I tried hard to make it pass windows CI but the libsndfile dependency is quitte messy.

Reply:
So we can ship `noarch: python` even if a package doesn't run on a specific platform due to a dependency issue. So as long as this package would run on windows if the deps were there, then we can go ahead and make it noarch. Does that make sense?

Reply:
I am ok with that, I tried to do as well with `musdb` recipe as it was suggested by another reviewer, but I was never able to make it pass through Win CI (lot of dependencies conflicted for unknown reason only under windows). Also the library is only available for `3.6` and `3.7` versions of Python (I could though apply selector to Python dependencies directly instead).

Reply:
Right. Use >= in the python requirements and then ignore failures on any platforms. It just has to build on Linux. We like noarch because it only builds on Linux and so saves us CI time.